go/cmd/gh-ost/main.go

```diff
@@ -51,7 +51,6 @@ func main() {
 	flag.StringVar(&migrationContext.AssumeMasterHostname, "assume-master-host", "", "(optional) explicitly tell gh-ost the identity of the master. Format: some.host.com[:port] This is useful in master-master setups where you wish to pick an explicit master, or in a tungsten-replicator where gh-ost is unable to determine the master")
 	flag.IntVar(&migrationContext.InspectorConnectionConfig.Key.Port, "port", 3306, "MySQL port (preferably a replica, not the master)")
 	flag.Float64Var(&migrationContext.InspectorConnectionConfig.Timeout, "mysql-timeout", 0.0, "Connect, read and write timeout for MySQL")
-	flag.Float64Var(&migrationContext.InspectorConnectionConfig.WaitTimeout, "mysql-wait-timeout", 0.0, "wait_timeout for MySQL sessions")
```

Comment:
Removing a flag makes this PR backwards incompatible, and will break `gh-ost` for any user supplying the flag.

Reply:
@shlomi-noach this flag was introduced just a few days prior in #1401 to solve the same root cause and no release has been cut yet with it

I later decided to isolate the `wait_timeout` change to cut-over time only (in this PR) to address the problem more closely and to avoid a larger change

That said, I think we could keep this flag around for more generic reasons

Reply:
Ah, cool, I did not realize the flag was only added recently. Remove it, then!